plenum/common/ledger_manager.py

```diff
@@ -291,6 +316,13 @@ def processLedgerStatus(self, status: LedgerStatus, frm: str):
             if self.isLedgerOld(ledgerStatus):
                 ledger_status = self.owner.build_ledger_status(ledgerId)
                 self.sendTo(ledger_status, frm)
+                if ledgerId not in self.consistency_proof_action_ids or \
+                        ledgerId in self.consistency_proof_action_ids and \
+                        self.consistency_proof_action_ids[ledgerId] is None:
```

Comment:
This condition will be simplified if `_cancel_request_ledger_statuses_and_consistency_proofs` is corrected (see the comment above).

Reply:
Done.